rest/_examples/monitor_history.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	api "gopkg.in/ns1/ns1-go.v2/rest"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/monitor"
+)
+
+// Helper that initializes rest api client from environment variable.
+func initClient() *api.Client {
```

Comment:
So this is a bit of an opinionated thing, but you could define client outside of the function and make this a `func init(){}`, that way it runs before main does without explicitly calling the function, it's a style thing more than anything else though, I'd say it's fine as is.